### PR TITLE
Fix dynamodb template for python not creating successfully.

### PR DESCRIPTION
**Pull request by zhangzhx**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Related Issue(s)
#1042 

## Testing
Added DynamoDB template UI test.

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 1]
```diff
@@ -266,6 +266,7 @@ sam.init.template.hello_world.description=A basic SAM app
 sam.init.template.dynamodb_cookiecutter.name=AWS SAM DynamoDB Event Example
 sam.init.template.dynamodb_cookiecutter.description=Sample SAM Template to interact with DynamoDB Events
 sam.init.error.no.project.basepath=Unable to determine project basepath
+sam.init.error.no.root_folder=Can't find the sam init root folder by SAM CLI.
```
Comment (abrooksv) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties]: capital SAM in both spots

Comment (zhangzhx) [reply resources/resources/software/aws/toolkits/resources/localized_messages.properties]: Done.

File: jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt [hunk 2]
```diff
@@ -48,6 +49,10 @@ object SamInitRunner {
             throw RuntimeException("${message("sam.init.execution_error")}: ${process.stderrLines.last()}")
         }
 
-        FileUtil.copyDirContent(tempDir.resolve(name), VfsUtil.virtualToIoFile(outputDir))
+        val rootFolder = tempDir.listFiles(FileFilter { it.isDirectory })?.firstOrNull()
+            ?: throw RuntimeException(message("sam.init.error.no.root_folder"))
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt]: Lets add the folder name for debugging

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 3]
```diff
@@ -266,6 +266,7 @@ sam.init.template.hello_world.description=A basic SAM app
 sam.init.template.dynamodb_cookiecutter.name=AWS SAM DynamoDB Event Example
 sam.init.template.dynamodb_cookiecutter.description=Sample SAM Template to interact with DynamoDB Events
 sam.init.error.no.project.basepath=Unable to determine project basepath
+sam.init.error.no.root_folder=Can't find the SAM init root folder by SAM CLI.
```
Comment (abrooksv) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties]: ... root folder in ''{0}''

File: jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt [hunk 4]
```diff
@@ -48,6 +49,10 @@ object SamInitRunner {
             throw RuntimeException("${message("sam.init.execution_error")}: ${process.stderrLines.last()}")
         }
 
-        FileUtil.copyDirContent(tempDir.resolve(name), VfsUtil.virtualToIoFile(outputDir))
+        val rootFolder = tempDir.listFiles(FileFilter { it.isDirectory })?.firstOrNull()
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt]: Assert the list is one

File: jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt (hunk 2 above)
Comment (zhangzhx) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt]: Done.

File: jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt (hunk 4 above)
Comment (zhangzhx) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt]: Done.

**Pull request merged**